KDIC/BlogViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  BlogViewController.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 12/27/15.
+//  Copyright © 2015 Colin Tremblay. All rights reserved.
+//  NOTE: XCode wont let me refector this class is actully for the Blog
+
+import UIKit
+import SafariServices
+
+class BlogViewController: UIViewController,  UITableViewDataSource, UITableViewDelegate {
+
+    @IBOutlet weak var blogPostTableView: UITableView!
+    
+    var blogsDict: NSArray?
+    
+    var kdicScrapperReqs: KdicScrapperReqs?
```

Comment:
causes compile error -  instance member blogPostTableView can not be used on type BlogViewControler